src/flexible_type/CMakeLists.txt

```diff
@@ -6,14 +6,14 @@ project(flexible_type)
 #/*                                                                        */
 #/**************************************************************************/
 
-make_library(flexible_type
+make_library(flexible_type OBJECT
   SOURCES
     flexible_type.cpp
     string_escape.cpp
     flexible_type_conversion_utilities.cpp
     flexible_type_spirit_parser.cpp
   REQUIRES
-    logger
-    image_type
-    EXTERNAL_VISIBILITY
-  )
+    logger		
+    image_type		
```

Comment:
Nit: I'm assuming this trailing whitespace is accidental?

Reply:
Thanks, good catch. Will fix.